src/common/media.cpp

```diff
@@ -51,51 +51,10 @@ media_state_t media_get_state(void) {
     return media_state;
 }
 
-void media_get_sfn_path(char *sfn, const char *filepath) {
```

Comment:
Why was this moved into GUI file browser? This is an operation of the file system, it has nothing to do with UI, it must be kept here.